tests/business-logic/User.test.ts

```diff
@@ -0,0 +1,86 @@
+import UserEntity from "@business-logic/User";
+
+import { hashPassword } from "@helpers/auth";
+import prisma from "@helpers/prisma";
+import { teardown } from "@helpers/tests/teardown";
+
+let hashedPassword: string;
+
+describe("User", () => {
+  beforeEach(async () => {
+    await teardown();
+    hashedPassword = await hashPassword("User@123!");
+  });
+
+  describe("#create", () => {
+    it("throws error if email address provided was invalid", async () => {
+      const payload = {
+        name: "John Doe",
+        email: "johndoegmail.com",
+        password: hashedPassword,
+      };
+      const entity = new UserEntity();
+      await expect(async () => {
+        await entity.create(payload);
+      }).rejects.toThrowError("Invalid email");
+    });
+    it("throws error if password provided was invalid", async () => {
+      const payload = {
+        name: "John Doe",
+        email: "johndoe@gmail.com",
```

Comment:
can you always use `...@example.com` for urls and email addresses in tests. just to make sure that we don't accidentally spam anyone

Reply:
Yeah sure, I'm now using [@example.com]